docs/scripts/make_dataset_zoo_docs.py

```
@@ -0,0 +1,177 @@
+            header=title,
+            description=description,
+            link_slug=link_slug,
+            image_rel=image_rel,
+            tags=tags,
+        )
+        card_items.append(item.strip() + "\n")
+
+    cards_path = out_dir / "dataset_cards.rst"
+    cards_path.write_text("\n".join(card_items), encoding="utf-8")
+
+    for slug, title, section_content in datasets:
```

Ruff (0.13.3)

Loop control variable `title` not used within loop body

(B007)